packages/server/tests/utils/TestUtils.ts

```diff
@@ -638,3 +638,22 @@ export function generateMpeWorkflowState(
         ...overrides,
     };
 }
+
+export function noop(): void {
+    return undefined;
+}
+
+export function createSpyOnClientSocketEvent<
+    Event extends keyof AllServerToClientEvents,
+>(
+    socket: TypedTestSocket,
+    event: Event,
+): sinon.SinonSpy<
+    Parameters<AllServerToClientEvents[Event]>,
+    ReturnType<AllServerToClientEvents[Event]>
+> {
+    const customSpy = sinon.spy<AllServerToClientEvents[Event]>(noop);
+
+    socket.on(event, customSpy as any); //tmp fix ?
```

Comment:
Instead of `as any`, which should never be used as it deactivates TypeScript compiler now and for all time, we should use `@ts-expect-error <Message indicating why we need that>`.
If the error disappears in the future, the compiler will throw an error because it is excepting an error.

This is not a crucial part of our codebase, so there is no need to change it. I just wanted to point out this detail. 😄